.github/workflows/review-testing-instructions.yml

```diff
@@ -0,0 +1,48 @@
+name: Remind reviewers to also review the testing instructions.
+on:
+  pull_request:
+    types: [review_requested]
+
+permissions: {}
+
+jobs:
+  add-testing-instructions-review-comment:
+    runs-on: ubuntu-20.04
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Get the username of requested reviewers
+        id: get_reviewer_username
+        run: |
+          # Retrieves the username of all reviewers and stores them in a comma-separated list
+          reviewers=$(echo '${{ toJson(github.event.pull_request.requested_reviewers[*].login) }}' | jq -r 'map("@\(.)") | join(", ")')
+          echo "REVIEWERS=$reviewers" >> $GITHUB_ENV
+
+      - name: Get the name of requested teams
+        id: get_team_name
+        run: |
+          # Retrieves the name of all teams asked for review and stores them in a comma-separated list
+          teams=$(echo '${{ toJson(github.event.pull_request.requested_teams[*].slug) }}' | jq -r 'map("@woocommerce/\(.)") | join(", ")')
+          echo "TEAMS=$teams" >> $GITHUB_ENV
+
+      - name: Find the comment by github-actions[bot] asking for reviewing the testing instructions
+        uses: peter-evans/find-comment@v2
+        id: find-comment
+        with:
+            issue-number: ${{ github.event.pull_request.number }}
+            comment-author: 'github-actions[bot]'
+            body-includes: please make sure to review the testing instructions
+
+      - name: Create or update PR comment asking for reviewers to review the testing instructions
+        uses: peter-evans/create-or-update-comment@v2
```

Comment:
Same for this one to pin to SHA
https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions